### ISSUE TITLE
Error on note import: 'NoneType' object has no attribute 'id'

### ISSUE THREAD

**Issue by ameer4** [djsudduth/keep-it-markdown#74]
I imported notes from iphone (as md files) using an app in mac app store.
After using keep-it-markdown, I encountered this error:

You've succesfully logged into Google Keep using local keyring access token!
Importing note: - from -.md
Label doesn't exist! - label: my_label  Use pre-defined labels when importing
Error on note import: 'NoneType' object has no attribute 'id'

any tips?

**Comment by djsudduth** [comment]
Looks like one of these possibilities:

1. You have an empty note with a blank name 
2. The settings.cfg file is missing the labels/tags you want in the note (you must use existing labels you have in Keep already - KIM won't create new labels)

Take a look at the README section on using the import -i flag for details with settings. Let me know what you discover - it could still be a bug on my end!

**Comment by djsudduth** [comment]
Any update @ameer4

**Comment by ameer4** [comment]
Sorry for late..
Yes, it was the label!
I created one, then added it in settings.cfg 

Then..
Encountered this error:

```
Importing note: - from -.md
Error on note import: 'utf-8' codec can't decode byte 0xb0 in position 37: invalid start byte
```
So I converted all *.md files using notepad++ python script.

it was a pain for me (who not used to coding) to know how to patch convert :) 

it may good idea if the tool auto-recognize >> auto-convert (or ignore the converted one) on the fly.

This is the script I used (source: https://gist.github.com/dogancelik/2a88c81d309a753cecd8b8460d3098bc   )

`from glob import glob
from Npp import notepad

globPath = "H:\keep-it-markdown-0.5.2\import\markdown_files\*.md"

for file in glob(globPath):
  notepad.open(file)
  notepad.runMenuCommand("Encoding", "Convert to UTF-8")
  notepad.save()
  notepad.close()`

since many *.md files was actually named like this: "._xxx.md" the script ignored these files, so I batch renamed (removed "._" from file names)

Now it successfully starts uploading md files to keep..

thank you for your tool.
